### ISSUE TITLE
No visual difference between Dropdown w/ and w/out disable-select

### ISSUE THREAD

**Issue by joshsylvester** [predixdesignsystem/px-dropdown#79]
There appears to be no visual difference between a `multi` dropdown and a `multi` dropdown that has `disable-select`, and between a normal dropdown, and one that has `disable-select`.

## Expected behavior and actual behavior:
A visual difference in the dropdown items that let the user know that the options are not selectable (possibly a different hover color, no hover at all, or greyed-out options?)

## Steps to reproduce the problem:
Compare visually the following (taken from the [example page](https://www.predix-ui.com/#/elements/px-dropdown)) two dropdowns:

**Multi-select**
```
<px-dropdown
  items='[{"key":"1","val":"iPhone"},{"key":"2","val":"Android"},{"key":"3","val":"Blackberry"},{"key":"4","val":"Windows Phone"},{"key":"5","val":"Flip Phone","disabled":true}]'
  sort-mode="key"
  button-style="default"
  display-value="Select"
  multi
  trigger-height="30">
</px-dropdown>
```
**Multi-select** w/ `disable-select`
```
<px-dropdown
  items='[{"key":"1","val":"iPhone"},{"key":"2","val":"Android"},{"key":"3","val":"Blackberry"},{"key":"4","val":"Windows Phone"},{"key":"5","val":"Flip Phone","disabled":true}]'
  sort-mode="key"
  button-style="default"
  display-value="Select"
  disable-select
  multi
  trigger-height="30">
</px-dropdown>
```

## Environment (_component version number, Browser (including version), operating system, hardware, etc_):
Predix-ui.com

**Comment by joshsylvester** [comment]
@randy - i've found a workaround for the visual style, thanks to @davidrleonard, with one remaining issue:

for a `px-dropdown`, that has the `disable-select` flag passed - the mouse cursor is showing a hand  icon over the menu items, even when they should not be clickable. It seems to me that if `disable-select` is turned on - that the default pointer might be better - as the hand "suggests" to the user that they can click. the disconunity goes even further - because if a single item is disabled (as in the example) the mouse cursor changes to `cursor: not-allowed`.

expected: 
```
.dropdown-option.dropdown--disable-select  {
  cursor: default;
}

// or
.dropdown-option[data-disable-select] {
  cursor: default;
}
```
<img width="953" alt="cursor_and_predix_design_system___dropdown" src="https://user-images.githubusercontent.com/30244294/37664100-df0a43b4-2c17-11e8-8c1a-5328f6a5d3b5.png">

**Comment by randyaskin** [comment]
The `disable-select` property was introduced for this use case, where the menu options are entirely clickable. They are actionable menu items, they're just not selectable. Can you explain your application use case? Why do you need another level between selectable and disabled?

![screen shot 2018-03-20 at 8 37 51 am](https://user-images.githubusercontent.com/2130613/37665208-02e3ed2e-2c1a-11e8-9375-47b01a47dd79.png)

**Comment by joshsylvester** [comment]
the use case we have is similar, but not entirely the same - yet I totally see where you were going with the current implementation. our use case is pretty well described in the original issue above - a dropdown that has items that need to be visible to the user - but they don't have permissions to change them. we've added two states (`readOnly` and `disabledViewable`) that we're controlling visually via style overrides, but we can't change the mouse-cursor along with said style override.

could a property like `disable-hover` be introduced, (along with the css above) which would solve both the hover and mouse cursor issue?

Take the new data grid as an example - https://www.predix-ui.com/#/elements/data-grid/px-data-grid let's say that the current user has access to see a table, but not to edit which fields are shown. the "Actions" menu (top right to the table) should be there - but the dropdown would have `disable-select` letting them see the items in the dropdown, but showing them visually) that they don't have access to edit those items. In our UI, we wanted the additional visual cues to show the whole dropdown as disabled (including the button & items) while still allowing the user to see the options that were previously selected (in a `multi` config)

![screen_shot_2018-03-07_at_11 05 05_am](https://user-images.githubusercontent.com/30244294/37665884-785370e2-2c1b-11e8-9ce8-875384ad9854.png)

**Comment by arongraham-ge** [comment]
released in version v4.7.10
